internal/commands/loglevel/unset.go

```diff
@@ -50,21 +49,23 @@ func unsetDebuggingConfiguration(ctx context.Context, component string) error {
 	}
 
 	if found {
-		return updateLogLevelConfiguration(ctx, spec)
+		return updateLogLevelConfiguration(ctx, spec, namespace, clusterName)
 	}
 	return nil
 }
 
 // Unset resets log verbosity for a given component
 func Unset(ctx context.Context, args []string) error {
 	doc := `Usage:
-  sveltosctl log-level unset --component=<name>
+  sveltosctl log-level unset --component=<name> --namespace=<namespace> --cluster=<cluster-name>
```

Comment:
same comments as above:

[--namespace=] [--cluster=] cause those are optional
add cluster-type
indentation for the comment of each option should be same